### PR TITLE
AMITA_Assignment_25_Done_10mins

### PR DESCRIPTION
**Pull request by AmitaRJun20**
AMITA_Assignment_25_Done...Please Review...
SumofnumbersinString

### CODE REVIEW

File: src/amitaRJun20/SumOfAllNumbersInString.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+public class SumOfAllNumbersInString {
+	
+	int sum = 0;
+	
+	//method to display sum of all numbers in input String
+	void displaySum(String input) {
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				int number = Integer.parseInt(String.valueOf(input.charAt(index)));
```
Comment (MaheshKJun20) [inline src/amitaRJun20/SumOfAllNumbersInString.java]: We can avoid creating number variable and directly add this value using the expression

**Review by MaheshKJun20 — APPROVED** [review]
Hi Amit, 
Code looks good. Approving it for further review.
Have added a suggestion at one place you can check on that

File: src/amitaRJun20/SumOfAllNumbersInString.java (hunk 1 above)
Comment (AmitaRJun20) [reply src/amitaRJun20/SumOfAllNumbersInString.java]: updated as per suggestion...Thank you for feedback..

**Review by KrishnaKTechnocredits — APPROVED** [review]
it looks good to me.

**Pull request merged**